metrotranscom/doorway
# fix: script runner for new reserved community type (#4380)

This PR addresses #853 and pulls in the changes from core


## Description

Please include a summary of the change and which issue(s) is addressed.

## How Can This Be Tested/Reviewed?

* Reseed the backend and start it up
* Log in as a admin user (either through the partner UI or directly in the swagger doc)
* In the swagger doc http://localhost:3100/api#/scriptRunner/createNewReservedCommunityType run the script using the following data:
id : jurisdiction id (need to get it from the db)
name: "schoolEmployee"
description: "Employee of the school district"
* The new community type should be in the database and also selectable in the partner UI
